libtonode-tests/tests/concrete.rs

```diff
@@ -404,7 +406,7 @@ mod fast {
     }
 
     #[tokio::test]
-    async fn unspent_notes_are_not_saved() {
```

Comment:
This test actually shows that the `spend_pending_transaction` is not saved as well, does it not?   Or, at least, it strongly implies it with the popped transaction check at the end.